src/components/OfferMyAccountCard/OfferMyAccountCard.tsx

```diff
@@ -257,44 +320,15 @@ const OfferMyAccountCard = ({ offerId }: OfferMyAccountCardProps) => {
         <SubBoxStyled>
           <IconComponent />
           <SubBoxContentStyled>
-            <BoxTextStyled
-              dangerouslySetInnerHTML={{
-                __html: getDescription() || ''
-              }}
-            />
+            <BoxTextStyled>{getDescription()}</BoxTextStyled>
```

Comment:
please extract the result of invoking `getDescription` to a variable and reuse. we don't need to rerun this function (the check in line #319)